CHANGELOG.md

```diff
@@ -33,6 +33,7 @@ Fixes:
 - [#1239](https://github.com/rails-api/active_model_serializers/pull/1239) Fix duplicates in JSON API compound documents (@beauby)
 - [#1214](https://github.com/rails-api/active_model_serializers/pull/1214) retrieve the key from the reflection options when building associations (@NullVoxPopuli, @hut8)
 - [#1358](https://github.com/rails-api/active_model_serializers/pull/1358) Handle serializer file paths with spaces (@rwstauner, @bf4)
+- [#1352](https://github.com/rails-api/active_model_serializers/pull/1352) Isolate Rails-specifc code in Railties (@dgynn, @bf4)
```

Comment:
Should new changelogs be added on top or at the bottom? (Real question, I have no answer)